chat-plugins/prowrestling.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * Professional Wrestling Room Plugin
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This handles submitting questions for the podcast that the room runs periodically,
+ * instead of having to use another source to ask questions.
+ *
+ * @license MIT license
+ */
+'use strict';
+
+const FS = require('../fs');
+
+const PW_DATA_PATH = 'config/chat-plugins/prowrestling-questions.json';
+const MAX_QUESTION_LENGTH = 500;
+
+class ProWrestling {
+	constructor() {
+		this.pwData = Object.create(null);
+
+		this.loadQuesions = (() => {
+			try {
+				this.pwData = require(`../${PW_DATA_PATH}`);
+			} catch (error) {
+				if (error.code === 'MODULE_NOT_FOUND') {
+					FS(PW_DATA_PATH).write(JSON.stringify(this.pwData));
```

Comment:
It's not necessary to force the file to exist like this, since that'll happen once `this.saveQuestions` is called